src/python/geoclaw/topotools.py

```diff
@@ -796,6 +797,15 @@ def write(self, path, no_data_value=None, topo_type=None, masked=True):
         from data in Z.  The rest of the arguments are used to write the header
         data.
 
+        :Input:
+         - *path* (str)  - file to write
+         - *no_data_value* - values used to indicate missing data
+         - *topo_type* (int) - GeoClaw format topo_type 
+         - *masked* (bool) - unused??
+         - *header_order* (int) - indicates format of header lines
+             1 ==> write value then label  (original Fortran assumption)
+             2 ==> write label then value  (needed for .asc files in ArcGIS)
```

Comment:
Can I suggest we make this a string parameter just in case there are other headers we may want to use beyond ARCGIS and what we have been doing up until now?